internal/service/cloudbackupschedule/resource_cloud_backup_schedule_test.go

```diff
@@ -300,15 +307,15 @@ func TestAccBackupRSCloudBackupSchedule_copySettings(t *testing.T) {
 		CheckDestroy:             checkDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: configCopySettings(projectID, clusterName, false, &admin.DiskBackupSnapshotSchedule{
+				Config: configCopySettings(clusterInfo.ClusterTerraformStr, projectID, clusterInfo.ClusterResourceName, false, &admin.DiskBackupSnapshotSchedule{
 					ReferenceHourOfDay:    conversion.Pointer(3),
 					ReferenceMinuteOfHour: conversion.Pointer(45),
 					RestoreWindowDays:     conversion.Pointer(1),
 				}),
 				Check: resource.ComposeAggregateTestCheckFunc(checksCreate...),
 			},
 			{
-				Config: configCopySettings(projectID, clusterName, true, &admin.DiskBackupSnapshotSchedule{
+				Config: configCopySettings(clusterInfo.ClusterTerraformStr, projectID, clusterInfo.ClusterResourceName, true, &admin.DiskBackupSnapshotSchedule{
 					ReferenceHourOfDay:    conversion.Pointer(3),
```

Comment:
it might be clearer to have const for these values

Reply:
+1
nit: also now seems kinda redundant with 'cluster' in `clusterInfo.ClusterTerraformStr`, might be good to rename `ClusterTerraformStr` to `TerraformStr`, same for other attributes

Reply:
@maastha good idea. I'll do it as part of https://jira.mongodb.org/browse/CLOUDP-260997